.vscode/settings.json

```diff
@@ -24,10 +24,90 @@
     ],
     "python.testing.unittestEnabled": false,
     "cSpell.words": [
-        "furo",
+        "adic",
+        "arccos",
+        "arccosh",
+        "arcsin",
+        "arcsinh",
+        "arctan",
+        "arctanh",
+        "Bejger",
+        "bigcup",
+        "cachefunc",
+        "charpoly",
+        "classmethod",
+        "clopen",
+        "codim",
+        "codomain",
+        "coframe",
+        "coframes",
         "Conda",
+        "cputime",
+        "cysignals",
+        "Cython",
+        "d'Alembertian",
+        "dalembertian",
```

Comment:
```suggestion
```

Reply:
It's used like this in a few method/variable names, which I think is okay.